gigareview/docker-compose.yml

```diff
@@ -2,10 +2,34 @@ version: '3.7'
 
 services:
 
+  web:
+    image: nginx:latest
+    ports:
+      - "9170:80"
+    volumes:
+      - ./:/app
+      - ./default.conf:/etc/nginx/conf.d/default.conf
+
+  application:
```

Comment:
Hi @pli888,
container services ``application`` should not exit.
Since this is an Yii2 advanced template, there are two application servers: ``frontend`` and ``backend`` (respectively called ``public and  ``api`` in our config here). 
The ``public` is the one to be used for public access by web user, so that's the one that should serve the manuscript dashboard.